src/open_inwoner/cms/cases/tests/test_contactform.py

```diff
@@ -39,6 +42,9 @@
 
 
 @requests_mock.Mocker()
+@patch.object(
+    ContactmomentenClient, "retrieve_objectcontactmomenten_for_zaak", autospec=True
+)
```

Comment:
This looks odd; in most cases we mock API responses through the request mocker instead of mocking the API client. 

Also why is it here at class level? Does every test function need this? It adds a lot of noise to the signatures but in many of the test functions I don't see the mock object being used or asserted.

If it is needed to get an empty list response then please add an explicit `return_value=[]` to the patch so it is more clear, and decide if this really has to be at class level.

Reply:
I asked if it could be moved to the class level, because it seemed like none of these tests actually test this behavior (they test the contactform instead) and we just want to mock it out for all tests in this class

Reply:
Ok fine, but it is kinda noisy to have to boilerplate this on every test function, while the empty mock would be part of the mocks, especially if it isn't even important to the tests.